app/config/config.go

```diff
@@ -35,6 +39,21 @@ type Conf struct {
 		FilesLocation   string             `yaml:"files_location"`
 		RSSLocation     string             `yaml:"rss_location"`
 	} `yaml:"youtube"`
+
+	Telegram struct {
+		Server  string        `yaml:"server"`
+		Channel string        `yaml:"channel"`
```

Comment:
this one is unclear. Used to be a part of single-feed setup, but if we drop it chanel is irrelevant here, as they defined per feed

Reply:
My mistake, thanks for catching this.